### DISCUSSION TITLE
How do I install this software on a Mac?

### DISCUSSION THREAD

**Discussion by CorrosionInstruments** [jankae/LibreVNA#258] [General]
I can only find instructions for Windows and Ubuntu. Thanks

**Comment by jankae** [comment]
I don't have any Mac myself and can't really help much here. There are binaries available for Mac (latest build is this: https://github.com/jankae/LibreVNA/actions/runs/9340105482). You may need to install Qt or maybe the libraries are already included in the binary - I am not exactly sure how the process works for Mac.
